hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/procedures/ValidateMetadataTableFilesProcedure.scala

```diff
@@ -44,7 +44,7 @@ class ValidateMetadataTableFilesProcedure() extends BaseProcedure with Procedure
     StructField("partition", DataTypes.StringType, nullable = true, Metadata.empty),
     StructField("file_name", DataTypes.StringType, nullable = true, Metadata.empty),
     StructField("is_present_in_fs", DataTypes.BooleanType, nullable = true, Metadata.empty),
-    StructField("is_resent_in_metadata", DataTypes.BooleanType, nullable = true, Metadata.empty),
+    StructField("is_present_in_metadata", DataTypes.BooleanType, nullable = true, Metadata.empty),
```

Comment:
Are there any tests for it?

Reply:
This is the header of result set. Modify the wrong English spelling to better understand. 
There are no test cases here.